Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/SMGs/mp5_smg.yml

```diff
@@ -117,5 +117,21 @@
     zeroOnlyOnEmpty: true
   - type: Appearance
 
+- type: entity
+  parent: CMCartridge10x20mm
+  id: CMCartridge9mmSMG
+  name: cartridge (9mm SMG)
+  description: A 9mm cartridge for the mp5 submachinegun.
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - CMCartridge9mmSMG
+  - type: CartridgeAmmo
+    proto: Bullet10x20mm
```

Comment:
It's a little silly that the "9mm" ammo types are not interchangeable, but whatever, maybe they're different loads or some shit. It just baffles me that this is fundamentally just a renamed 10x20 round that can't fit in the other m63 or vice versa, even though the mp5 is statistically worse overall with a 1.2 damage modifier on the m63... and I'm starting to wonder if it would have just made more sense to leave it using the m63 ammo.

YIPPIEEE PARITY MAJOR